service/worker/batcher/workflow.go

```diff
@@ -304,7 +333,23 @@ func startTaskProcessor(
 
 			switch batchParams.BatchType {
 			case BatchTypeTerminate:
-				err = processTerminateTask(ctx, limiter, task, batchParams, client)
+				err = processChildRecursiveTask(ctx, limiter, task, batchParams, client,
```

Comment:
nit: can just be named `processTask` now..

Reply:
Makes sense. Thank you so much for those useful feedbacks to improve the code quality. Learned a lot!